modules/catalog-dsl/build.gradle

```diff
@@ -26,6 +26,7 @@ dependencies {
     implementation libs.jetbrains.annotations
     implementation project(':ignite-api')
     implementation project(':ignite-core')
+    implementation project(':ignite-catalog')
```

Comment:
Please remove, this brings a lot of extra dependencies to `ignite-client`, such as `ignite-configuration` and `ignite-vault` - not acceptable.